src/zope/tales/tales.py

```diff
@@ -20,7 +20,8 @@
 
 try:
     from html import escape
-except ImportError:
+except ImportError:  # pragma: no cover
+    # PY2
     from cgi import escape
```

Comment:
Have I ever mentioned https://pypi.org/project/coverage-python-version/ to you?

Reply:
Sorry, I always forget about this handy tool.